ckanext/data_qld/pylons_plugin.py

```diff
@@ -22,7 +22,11 @@ def before_map(self, m):
         m.connect(
             'data_qld.show_schema',
```

Comment:
This route name should probably be changed to 'data_qld.show_resource_schema'.

Reply:
+

Reply:
Will be fixed and pushed by Alex shortly.